test/python/pulse/test_reschedule.py

```diff
@@ -168,9 +169,21 @@ def test_dont_add_all(self):
         acquired_qubits = set()
         for _, inst in sched.instructions:
             if isinstance(inst, AcquireInstruction):
-                acquired_qubits.update({a.index for a in inst.acquires})
+                acquired_qubits.add(inst.acquire.index)
         self.assertEqual(acquired_qubits, {0, 1, 2, 3})
 
+    def test_explicitly_non_acquired_qubit(self):
+        """Test for time and duration of qubit that was not acquired, but in meas map."""
+        sched = add_implicit_acquires(self.sched, [[0, 5]])
+        times = set()
+        durations = set()
+        for time, inst in sched.instructions:
+            if isinstance(inst, AcquireInstruction):
+                times.add(time)
+                durations.add(inst.duration)
+        self.assertEqual(times, {0, 5})
```

Comment:
Is this right? It seems to me that all the acquires should happen at t=5 only, and that the duration should also be 5 only.

Reply:
fixed